src/System.Private.ServiceModel/tests/Scenarios/SelfHostWcfService/WcfService.cs

```diff
@@ -234,5 +236,46 @@ public XmlCompositeType GetDataUsingXmlSerializer(XmlCompositeType composite)
             composite.StringValue = composite.StringValue;
             return composite;
         }
+
+        public Stream GetStream(string data)
+        {
+            return StringToStream(data);
+        }
+
+        public bool UploadStream(Stream stream)
+        {
+            string data = StreamToString(stream);
+
+            if (!string.IsNullOrWhiteSpace(data))
+            {
+                return true;
```

Comment:
Careful... if you return `data` back as a string, you're not going to be testing StreamedResponse. Depends what your intention for the test is :smile: